mhs/common/mhs_common/workflow/sync_async.py

```diff
@@ -93,16 +94,16 @@ async def _retrieve_async_response(self, message_id, wdo: wd.WorkDescription):
                          fparams={'messageId': message_id})
             return 500, "No async response received from sync-async store"
 
-    async def handle_inbound_message(self, message_id: str, correlation_id: str, work_description: wd.WorkDescription,
-                                     payload: str):
+    async def handle_inbound_message(self, message_id: str, correlation_id: str, work_description: wd.WorkDescription, message_data: MessageData):
         logger.info('Entered sync-async inbound workflow')
         await wd.update_status_with_retries(work_description,
                                             work_description.set_inbound_status,
                                             wd.MessageStatus.INBOUND_RESPONSE_RECEIVED,
                                             self.persistence_store_retries)
 
         try:
-            await self._add_to_sync_async_store(message_id, {CORRELATION_ID: correlation_id, MESSAGE_DATA: payload})
+            #TODO: check if need to add attachments and manifest as well
```

Comment:
Is this TODO added to JIRA?

Reply:
I forgot to remove it. Line 106 has everything in `message_data`